common/src/test/java/io/honeycomb/opentelemetry/sdk/trace/samplers/DeterministicTraceSamplerTest.java

```diff
@@ -1,24 +1,26 @@
 package io.honeycomb.opentelemetry.sdk.trace.samplers;
 
-import java.util.Collections;
-
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.Assertions;
-import io.opentelemetry.sdk.trace.IdGenerator;
-import io.opentelemetry.sdk.trace.samplers.Sampler;
-import io.opentelemetry.sdk.trace.samplers.SamplingDecision;
-import io.opentelemetry.sdk.trace.samplers.SamplingResult;
 import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.api.trace.SpanKind;
 import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.trace.IdGenerator;
+import io.opentelemetry.sdk.trace.data.LinkData;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import io.opentelemetry.sdk.trace.samplers.SamplingDecision;
+import io.opentelemetry.sdk.trace.samplers.SamplingResult;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.List;
 
 public class DeterministicTraceSamplerTest {
 
     @Test
     public void test_neverSample() {
         int sampleRate = 0;
-        Sampler sampler = new DeterministicTraceSampler(sampleRate);
+        Sampler sampler = new DeterministicTraceSampler(Sampler.alwaysOn(), sampleRate);
```

Comment:
I guess this isn't strictly needed as we have a constructor that just takes the sampleRate param. We exercise the the constructor in later tests to provide a decordatedSampler.